README.md

```diff
@@ -69,6 +74,14 @@ script_duration_seconds{script="success"} 5.013670
 script_success{script="success"} 1
 ```
 
+A `target` can be specified for each script in the configuration file, or, more
+usefully, passed as a parameter which is made available to the script(s) that
+gets run as an environment variable named `$TARGET`. This allows you to leverage
+Prometheus targets, if you happen to need the script to operate on an arbitrary
+number of remote hosts/services.
+
+`$ curl http://localhost:9172/probe?name=example.sh&target=service.example.com`
```

Comment:
Is the `name=` parameter correct based on the earlier example? I expected this to be `name=target`. Maybe a less ambiguous name here would be helpful. e.g. `name=script_with_parameter&target=service.example.com`.